Update enable-tag-inheritance.md (#124570)
Azure/azure-content: articles/cost-management-billing/costs/enable-tag-inheritance.md

## Usage record updates

After the tag inheritance setting is updated, it takes about 8-24 hours for the child resource usage records to get updated. Any update to the setting or the tags being inherited takes effect for the current month.

> As an example:  If the tag inheritance setting is enabled on October 20, child resource usage records are updated from October 1 using the tags that existed on October 20. 

Reviewer: ```suggestion
For example, if the tag inheritance setting is enabled on October 20, child resource usage records are updated from October 1 using the tags that existed on October 20. 
```
Minor edit
